Makefile

```diff
@@ -82,176 +70,104 @@ $(MYGOBIN)/pluginator:
 	cd cmd/pluginator; \
 	go install .
 
+
+# --- Build targets ---
+
 # Build from local source.
 $(MYGOBIN)/kustomize: build-kustomize-api
 	cd kustomize; \
 	go install .
 
-.PHONY: install-tools
-install-tools: \
-	$(MYGOBIN)/goimports \
-	$(MYGOBIN)/golangci-lint-kustomize \
-	$(MYGOBIN)/gorepomod \
-	$(MYGOBIN)/helmV3 \
-	$(MYGOBIN)/k8scopy \
-	$(MYGOBIN)/mdrip \
-	$(MYGOBIN)/pluginator \
-	$(MYGOBIN)/stringer
+kustomize: $(MYGOBIN)/kustomize
 
-### Begin kustomize plugin rules.
```

Comment:
This entire section was just lifted as-is into `Makefile-plugins.mk` to make this file easier to navigate.